appveyor.yml

```diff
@@ -1,8 +1,12 @@
-before_build:
+install:
+  - npm install -g mocha
   - npm install
+  - npm link
```

Comment:
This is a good point, I need to look more into this.  In hindsight this could be considered a bit of a hack so i'll clean it up.  Previously this used the local mocha install and 'mocha-appveyor' as the reporter name.  Now it uses a global mocha install and 'mocha-appveyor-reporter' (the name of the repo, how people need to address it in their projects).   I'll revert this back to using `./node_modules/.bin/mocha` so this step is no longer necessary.